tools/ctl/ide/explorer/explorer.go

```diff
@@ -206,46 +206,34 @@ func diffArtifacts(ctx context.Context, butler localfiles.Butler, example rundex
 	}
 	var rba, usa string
 	{
-		var err error
-		if example.WasSmoketest() {
-			rba, err = butler.Fetch(ctx, example.RunID, example.WasSmoketest(), rebuild.DebugRebuildAsset.For(t))
+		dir, err := os.MkdirTemp("", "*")
```

Comment:
i'd mention that we're now stabilizing smoketest resources which, AFAIR, are already stabilized. i think this is fine behavior but I'd like to leave some indication of it on the commit / PR.